plugins/must-gather/skills/must-gather-analyzer/scripts/run-camgi.sh

```
@@ -0,0 +1,243 @@
+command_exists() {
+    command -v "$1" >/dev/null 2>&1
+}
+
+# Function to stop camgi containers
+stop_camgi() {
+    echo -e "${BLUE}Stopping camgi containers...${NC}"
+
+    # Check for podman
+    if command_exists podman; then
+        local containers
+        containers =$(podman ps -q --filter ancestor=quay.io/elmiko/okd-camgi)
```

Shellcheck (0.11.0)

[error] Remove spaces around = to assign (or use [ ] to compare, or quote '=' if literal).

(SC2283)